shtdlib.sh

```diff
@@ -1682,8 +1684,36 @@ function strip_space {
     echo -n "${@}" | sed -e 's/^[[:space:]]*//' -e 's/[[:space:]]*$//'
 }
 
-# Usage
-# This function is used to safely edit files for config parameters, etc
+# Load ini file parameter
+# Requires at least two arguments but optionally accepts a third, ini_section
+# If ini_section is specified and multiple sections match an error will be
+# raised. Multiple matches will otherwise be returned
```

Comment:
`s/matches/parameter matches/`?

Reply:
Unless this sentence is related to the one before about multiple INI sections

Reply:
Clarified